content/contributing/documentation/rst_cheat_sheet.rst

```diff
@@ -211,6 +211,48 @@ Use the `command` markup to highlight a command.
           Run the command :command:`make clean html` to delete existing built files and build the
           documentation to HTML.
 
+.. _contributing/icons:
+
+Icons
+-----
+
+Use the `icon` markup to add a class name of an icon. The icon set used is *Font Awesome*. It is
+recommended to accompany an icon with a label (`guilabel`), however, it is not mandatory.
```

Comment:
maybe refer to "GUI element" with a ref link to `contributing/markups/guilabel` instead of "label (`guilabel`)"?